profile.d/toolbox.sh

```diff
@@ -5,8 +5,13 @@ toolbox_config="$HOME/.config/toolbox"
 host_welcome_stub="$toolbox_config/host-welcome-shown"
 toolbox_welcome_stub="$toolbox_config/toolbox-welcome-shown"
 
-if [ -f /run/ostree-booted ] \
```

Comment:
Umm... I think that we need to retain the check for `/run/ostree-booted` in addition to the new ones that you added, otherwise the blurb will show up on legacy Workstation too. I haven't actually checked because I don't have a Workstation system at hand, only Silverblue.

Things have been complicated by the fact that none of the stable versions of Silverblue that have shipped so far had *silverblue* as the `VARIANT_ID`.

Reply:
I took the liberty to tweak the patch myself.